barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy.test.cpp

```diff
@@ -41,9 +41,7 @@ template <typename Flavor> class ProtoGalaxyTests : public testing::Test {
     static void construct_circuit(Builder& builder)
     {
         if constexpr (IsGoblinFlavor<Flavor>) {
-            GoblinMockCircuits::construct_arithmetic_circuit(builder);
```

Comment:
`GoblinMockCircuits::construct_simple_circuit` calls `MockCircuits::construct_arithmetic_circuit` and adds some ECC ops.